### DISCUSSION TITLE
Alexa Media funziona solo per qualche minuto

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#3253] [HUB: Home Assistant]
Sono con HA su Cloudflare ma da tempo non riesco a far funzionare più Alexa. Nel mio caso non parla più ma riceve i comandi per attivare script e automazioni. Se chiedo di spegnere una luce lo fa ma tutti i comandi da interfaccia HA e gli script che dovrebbero fornire risposte locali (tipo: "ho acceso il boiler") non funzionano. Se fosse un problema di collegamento a Amazon teoricamente anche i comandi vocali non dovrebbero funzionare.
Ho letto che potrebbe essere un problema di Amazon.it. Qualche riscontro?

**Comment by Scialla** [comment]
Occhio a non confonderti, sono due cose differenti; per comandare i tuoi device da alexa userai Nabu casa oppure HAASKA e AWS, per avere le notifiche vocali usi Alexa Media Player.

Quest'ultimo sta avendo diversi problemi ultimamente come ampiamente discusso ovunque, anche qui.
Prova a modificare il tempo di scansione nelle opzioni e mettilo (da 60) a 600

**Reply by NuccioT** [reply]
Grazie, ora provo. Quindi AWS e HAASKA funzionano. Quello che non funziona è solo AMP su HA. Te lo chiedo perché così evito di andare a lavorare su AWS.

**Comment by NuccioT** [comment]
Provato ma non è cambiato nulla. Dopo qualche minuto salta. Ora mi dà entità non disponibile sul dashboard.

**Reply by Scialla** [reply]
Sei alla 4.13.4?

Se, si, dalle opzioni disabilita il polling

**Reply by NuccioT** [reply]
ok, grazie. Con polling disabilitato sembra funzionare. Pensi che ci vorrà ancora molto per una versione stabile?

**Comment by NuccioT** [comment]
Ho uno script per far dire ad alexa il nome dell'ultimo audio riprodotto:
```alias: Casa - P3-01) - Interrogazione ultimo file riprodotto
sequence:
  - data: {}
    action: alexa_media.update_last_called
  - data:
      data:
        type: tts
      message: >-
        "Ultimo audio riprodotto: {{ states("input_text.ultimo_file_riprodotto")
        }}."
      target: null
    action: notify.alexa_media_last_called
mode: single
icon: mdi:volume-high
```

attivando questo script la connessione con alexa salta e devo ricaricare l'integrazione

Se invece non chiamo l'update last called la connessione rimane attiva
 
```
alias: test alexa
sequence:
  - action: notify.alexa_media_last_called
    metadata: {}
    data:
      message: >-
        "Ultimo audio riprodotto: {{ states("input_text.ultimo_file_riprodotto")
        }}."
      data:
        type: tts
description: ""
```

Non so se questo possa essere utile a risolvere il problema che ormai sta scoraggiando l'utilizzo di alexa (almeno per quanto mi riguarda)

**Reply by Scialla** [reply]
Fa la stessa cosa che fa il polling.. il problema non è risolvibile da noi

**Comment by KC-inDomus** [comment]
l'ultimo aggiornamento risolve il problema.
